Makefile

```diff
@@ -0,0 +1,40 @@
+# define the name of the virtual environment directory
+VENV := venv
+
+# default target, when make executed without arguments
+all: venv
```

Comment:
Why do we have no rule to make `venv`, but it's the default target?

I'd suggest a rule like:

```
venv:
  python3 -m venv ${VENV}
```

Reply:
yeah I'll add it back
- [ ] add it back